client/signup/steps/plans/index.jsx

```diff
@@ -252,6 +252,10 @@ export class PlansStep extends Component {
 	getHeaderText() {
 		const { headerText, translate, eligibleForProPlan, locale } = this.props;
 
+		if ( headerText ) {
+			return headerText;
+		}
+
```

Comment:
This change is changing the priority of the headerText prop. Is this intended? the behaviour now is different than before the change.

If eligibleForProPlan is true, and header has some value, we will never show the translations for eligibleForProPlan

Reply:
See [this comment](https://github.com/Automattic/wp-calypso/pull/67220#discussion_r959743520) for clarification